Gemfile

```diff
@@ -2,8 +2,8 @@ source "https://rubygems.org"
 
 ruby "2.2.1"
 
-gem 'activesupport', '~> 4.1.4'
-gem 'activerecord', '~> 4.1.4'
+gem 'activesupport', '4.2.1'
```

Comment:
Is this the equivalent of `=4.2.1` or `~>4.2.1`? I'd prefer locking to 4.2.x rather than 4.2.1.